### PR TITLE
First draft of community process

### PR DESCRIPTION
**Pull request by jroper**

### CODE REVIEW

File: public/markdown/community-process.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+* Making it clear and transparent to people who are new to the Play community how decisions are made, who makes those decisions, and how new people can attain any decision making responsibility.
+* Providing a concrete definition of the decision making process in Play, so that it can be referred to and improved upon, should the need arise.
+
+## Project ownership
+
+The Play project source code is owned and licensed by [Typesafe](https://www.typesafe.com).  Typesafe employees a team of developers who work on Play full time, and has the final say on both product decisions and technical decisions for the project.
+
+Though Typesafe wholely owns the Play project and has the final say in any decisions made, Typesafe does not own the Play community, nor does Typesafe take for granted its existence.  The Play community is arguably just as important, if not more important, than the Play project itself.  With over 400 community contributors to the core of Play, and hundreds more to the broader Play ecosystem, the Play community could easily survive without or in spite of Typesafe, if it ever came to that.
```
Comment (richdougherty) [inline public/markdown/community-process.md]: Typo: s/wholely/wholly/

File: public/markdown/community-process.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+## Decision making
+
+Decisions in the Play project fall in two broad categories:
+
+* Implementation decisions, this includes whether a pull request is up to standard to be merged, how a feature or improvement should be implemented.
+* Design and house keeping decisions, aka everything else.  This includes major design decisions, road maps, release schedules, decisions about how the project should be run and managed, what tools should be used, etc.
+
+### Design and house keeping decisions
+
+The primary place for discussion about the design of Play and how the Play project is run is the [play-framework-dev](https://groups.google.com/forum/#!forum/play-framework-dev) mailing list.  All major new features, refactorings or changes to the project should first be discussed in this forum.  The aim of the discussions is to reach an understanding on whether the task will be done, and how it will be done.  When a new topic is posted, interested parties are encouraged to comment with their affirmation or concerns.
```
Comment (richdougherty) [inline public/markdown/community-process.md]: I don't do a very good job of using the dev mailing list ot communicate what I'm working on. I'll try and do more talking there.

File: public/markdown/community-process.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+While Typesafe ultimately has the last word on all decisions here, as much as possible we will endeavor to reach a concensus in the majority of the community.
```
Comment (richdougherty) [inline public/markdown/community-process.md]: Typo: s/concensus/consensus/

File: public/markdown/community-process.md (hunk 1 above)
Comment (jroper) [inline public/markdown/community-process.md]: Is that really how to spell it, or do you just love your daughter so much that you want to see her name everywhere?

File: public/markdown/community-process.md (hunk 3 above)
Comment (richdougherty) [inline public/markdown/community-process.md]: This typo occurs a few times below as well.

File: public/markdown/community-process.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+Implementation decisions primarily happen in pull requests.  They are initiated by the pull request itself, and through reviews and iteration, a concensus is formed for how the given change should be implemented.
+
+All interested parties are encouraged to involve themselves in reviewing pull requests and contributing to review discussions.
+
+The amount of concensus required for whether a pull request is merged or not depends on how much impact the pull request has.  For trivial changes, such as corrections to documentation, an integrator may simply merge it with no feedback from any other integrators.  For larger changes, at least one person who is familiar with the part of the code being modified should review it, preferably more.  For big refactorings, the pull request should be reviewed by at least 2 or 3 other integrators before it is merged.
+
+Whether a pull request is merged or not depends on many factors, including:
+
+* Appropriate level of test coverage and documentation, where necessary
+* Adherance to coding standards and other code quality factors
```
Comment (richdougherty) [inline public/markdown/community-process.md]: Typo: s/adherance/adherence/

File: public/markdown/community-process.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+Typesafe may also select integrators from its own staff.
+
+If an integrator stops contributing regularly to Play, their write access may be removed, though their membership in the Play GitHub organisation will still be maintained.
+
+## Rules for integrators
+
+All integrators should follow the processes outlined on this page, and should be an example to follow with regards to Play's [code of conduct](/conduct) and [contributor guidelines](/contributing).  There are also a few specific rules, outlined below:
+
+* An integrator must never push directly to a repository in the GitHub organisation.  All contributions, no matter how small, must be made through pull requests.  The only exceptions to this are when backporting changes, and changes that arise from cutting a release.
+* Generally, pull requests should be made from integrators personal forks, not from branches push to repositories in the playframework GitHub organisation.
```
Comment (richdougherty) [inline public/markdown/community-process.md]: Typo: s/integrators/integrators'/

File: public/markdown/community-process.md (hunk 1 above)
Comment (richdougherty) [inline public/markdown/community-process.md]: Yeah, maybe we should drop the 'w' too… :)

**Comment by richdougherty** [comment]
@jroper, thanks a lot for writing this up.

File: app/services/github/ContributorsSummariser.scala [hunk 6]
```diff
@@ -76,6 +93,7 @@ class CachingContributorsSummariser @Inject() (actorSystem: ActorSystem,
       case Failure(t) => Logger.error("Unable to load contributors from GitHub", t)
       case Success(cs) =>
         if (contributors != cs) {
+          println(FallbackContributors.dumpContributors(cs))
```
Comment (yanns) [inline app/services/github/ContributorsSummariser.scala]: `println` is the new best practice for logging with Play? ;)

File: public/markdown/community-process.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+* Code reviews in GitHub
+* Raising, triaging and adding additional information to help resolve issues
+* Taking part in design and feature discussions on mailing lists
+* Answering questions on mailing lists and in stack overflow
+* Running, speaking at or contributing to user groups focussed on Play Framework
+* Speaking at conferences, blogging about or otherwise promoting Play Framework
+
+### Integrators
+
+An intergrator is anyone with write access to the source code and documentation of the Play project, or one of the projects that come under the [playframework](https://github.com/playframework) GitHub organization.  A current list of all integrators can be found on the [code and contributors](/code) page.
```
Comment (yanns) [inline public/markdown/community-process.md]: s/intergrator/integrator

File: app/services/github/ContributorsSummariser.scala (hunk 6 above)
Comment (jroper) [inline app/services/github/ContributorsSummariser.scala]: Oops...

**Comment by yanns** [comment]
:+1:

File: public/markdown/community-process.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+* Code reviews in GitHub
+* Raising, triaging and adding additional information to help resolve issues
+* Taking part in design and feature discussions on mailing lists
+* Answering questions on mailing lists and in stack overflow
+* Running, speaking at or contributing to user groups focussed on Play Framework
+* Speaking at conferences, blogging about or otherwise promoting Play Framework
+
+### Integrators
+
+An integrator is anyone with write access to the source code and documentation of the Play project, or one of the projects that come under the [playframework](https://github.com/playframework) GitHub organization.  A current list of all integrators can be found on the [code and contributors](/code) page.
```
Comment (mkurz) [inline public/markdown/community-process.md]: There are two spaces between these two sentences.

**Pull request merged**